arch/risc-v/src/common/riscv_internal.h

```diff
@@ -171,8 +160,8 @@ void riscv_ack_irq(int irq);
 void riscv_copystate(uint64_t *dest, uint64_t *src);
 void riscv_copyfullstate(uint64_t *dest, uint64_t *src);
 #else
-void riscv_copystate(uint32_t *dest, uint32_t *src);
-void riscv_copyfullstate(uint32_t *dest, uint32_t *src);
+void riscv_copystate(uintptr_t *dest, uintptr_t *src);
+void riscv_copyfullstate(uintptr_t *dest, uintptr_t *src);
 #endif
```

Comment:
```suggestion
void riscv_copystate(uintptr_t *dest, uintptr_t *src);
void riscv_copyfullstate(uintptr_t *dest, uintptr_t *src);
```
eliminate `#ifdef CONFIG_ARCH_RV64`

Reply:
Done